homeassistant/components/isy994/const.py

```diff
@@ -656,3 +656,38 @@
 HTTP_PORT = 80
 SCHEME_HTTPS = "https"
 HTTPS_PORT = 443
+
+INSTEON_RAMP_RATES = [
+    f"9 {UnitOfTime.MINUTES}",
+    f"8 {UnitOfTime.MINUTES}",
+    f"7 {UnitOfTime.MINUTES}",
+    f"6 {UnitOfTime.MINUTES}",
+    f"5 {UnitOfTime.MINUTES}",
+    f"4.5 {UnitOfTime.MINUTES}",
+    f"4 {UnitOfTime.MINUTES}",
+    f"3.5 {UnitOfTime.MINUTES}",
+    f"3 {UnitOfTime.MINUTES}",
+    f"2.5 {UnitOfTime.MINUTES}",
+    f"2 {UnitOfTime.MINUTES}",
+    f"1.5 {UnitOfTime.MINUTES}",
+    f"1 {UnitOfTime.MINUTES}",
+    f"47 {UnitOfTime.SECONDS}",
+    f"43 {UnitOfTime.SECONDS}",
+    f"38.5 {UnitOfTime.SECONDS}",
+    f"34 {UnitOfTime.SECONDS}",
+    f"32 {UnitOfTime.SECONDS}",
+    f"30 {UnitOfTime.SECONDS}",
+    f"28 {UnitOfTime.SECONDS}",
+    f"26 {UnitOfTime.SECONDS}",
+    f"23.5 {UnitOfTime.SECONDS}",
+    f"21.5 {UnitOfTime.SECONDS}",
+    f"19 {UnitOfTime.SECONDS}",
+    f"8.5 {UnitOfTime.SECONDS}",
+    f"6.5 {UnitOfTime.SECONDS}",
+    f"4.5 {UnitOfTime.SECONDS}",
+    f"2 {UnitOfTime.SECONDS}",
+    f"0.5 {UnitOfTime.SECONDS}",
+    f"0.3 {UnitOfTime.SECONDS}",
+    f"0.2 {UnitOfTime.SECONDS}",
+    f"0.1 {UnitOfTime.SECONDS}",
+]
```

Comment:
I think this might sense to construct in a loop at load time

Reply:
Ideally the numeric values come from pyisy

Reply:
I had me a nice fancy function to convert this from seconds in pyisy and spit it out--then I thought that was too much to do every time during runtime for values that will never change. I'll add it back in.

Reply:
Take a look at the latest commit and let me know if that's acceptable.

Reply:
LGTM assuming the intent is to drop it here

Reply:
Yep, wanted to make sure you were OK with the declaration. Will remove this.

Reply:
Done.